example/main.jsonnet

```diff
@@ -37,118 +37,149 @@ local obs = (import '../environments/base/observatorium.jsonnet');
 
 {
   local cr = self,
-  name:: 'observatorium-cr',
+  cr_no_tls_name:: 'observatorium-cr',
```

Comment:
Let's be consistent and use camelCase for everything

Reply:
Done